Meta/CMake/utils.cmake

```diff
@@ -221,3 +221,33 @@ function(download_file url path)
         endif()
     endif()
 endfunction()
+
+function(add_file_jakt source)
+    set(source ${CMAKE_CURRENT_SOURCE_DIR}/${source}.jakt)
+    get_filename_component(CMD_NAME_JAKT ${source} NAME_WE)
+    add_executable(${CMD_NAME_JAKT} "${CMD_NAME_JAKT}.cpp")
+    compile_jakt(${source})
+    target_link_libraries(${CMD_NAME_JAKT} LibC)
+    set_target_properties(${CMD_NAME_JAKT} PROPERTIES EXCLUDE_FROM_ALL TRUE)
+    install(TARGETS ${CMD_NAME_JAKT} RUNTIME DESTINATION bin OPTIONAL)
+    serenity_component(
+        ${CMD_NAME_JAKT}
+        RECOMMENDED
+        TARGETS ${CMD_NAME_JAKT}
+    )
+endfunction()
+
+function(add_directory_jakt directory)
```

Comment:
This looks much more reasonable. Though, this could be simplified further into one command that takes the jakt file and the executable name, instead of two commands with a bunch of copy paste.

e.g.
`function(add_jakt_executable exe_name source_file)`

Reply:
Yeah, here is a 3rd version like that.
Also added the idea of a YAKT_TARGETS list as a way of dealing with having a c++ and yakt version of something.

Reply:
Hmm. I'm not convinced we should add the JAKT_TARGETS workaround until we actually have some JAKT_TARGETS to add. Or at least, if we do keep it here in this PR, a comment explaining that this is the intended use should be used.

Reply:
oh I missed that you added hello-jakt to the list.

Ok, just a comment over it then :D then I think this is good to go. Thanks for adding a better helper!

Reply:
Sure, have added that.